spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -325,6 +340,64 @@
           end
         end
       end
+
+      context 'when searching for a specific keyword in result urls' do
+        context 'given there are 2 keywords with result urls contain the keyword vpn' do
+          it 'returns 2 keywords contains the matched urls' do
```

Comment:
```suggestion
          it 'returns 2 keywords contain the matched urls' do
```

Reply:
Fixed in https://github.com/nimblehq/ic-rails-phong-bliss/pull/78/commits/e91fb7286fde0e64ca3f24e6374149c1212aeebd